utils/build/docker/java/iast-common/src/main/java/com/datadoghq/system_tests/iast/utils/SqlExamples.java

```diff
@@ -27,6 +27,17 @@ public Object insecureSql(final String table, final StatementHandler handler) {
         }
     }
 
+    public void insecureSql(final String username, final String password, final StatementHandler handler) {
```

Comment:
Function not needed. See https://github.com/DataDog/system-tests/pull/1730

Reply:
Fixed